src/DssSpell.sol

```diff
@@ -43,8 +49,34 @@ contract DssSpellAction is DssAction {
     uint256 internal constant WAD     = 10 ** 18;
     uint256 internal constant RAY     = 10 ** 27;
 
+    function _sub(uint256 x, uint256 y) internal pure returns (uint256 z) {
+        require((z = x - y) <= x, "sub-underflow");
+    }
```

Comment:
0.8.16 now, we have safemath
```suggestion
```

Reply:
Thanks this was inherited from an old DC to 0 example, will remove